src/dbt_score/models.py

```diff
@@ -146,7 +146,8 @@ def _get_columns(
                     for test in test_values
                     if test.get("test_metadata", {})
                     .get("kwargs", {})
-                    .get("column_name")
+                    .get("column_name", "")
+                    .strip("`")
```

Comment:
```suggestion
                    .strip("`")  # BigQuery connector when "quote: true"
```
Question: is this a bug in dbt's BigQuery connector? If so, can we include a reference to an issue here?

Reply:
I couldn't find a related issue in dbt-adapters, and I am not sure if this is an unwanted behaviour from the connector.